src/helper/coordinate.rs

```diff
@@ -103,27 +103,27 @@ pub fn coordinate_from_index_with_lock(
 ) -> String {
     format!(
         "{}{}{}{}",
-        if is_lock_col == &true { "$" } else { "" },
+        if *is_lock_col { "$" } else { "" },
         string_from_column_index(col),
-        if is_lock_row == &true { "$" } else { "" },
+        if *is_lock_row { "$" } else { "" },
         row
     )
 }
 
 pub(crate) fn adjustment_insert_coordinate(num: &u32, root_num: &u32, offset_num: &u32) -> u32 {
-    let mut result = *num;
-    if (num >= root_num && offset_num > &0) || (num < root_num && offset_num < &0) {
-        result += offset_num;
+    if (num >= root_num && offset_num != &0) {
+        num + offset_num
+    } else {
+        *num
     }
-    result
 }
 
 pub(crate) fn adjustment_remove_coordinate(num: &u32, root_num: &u32, offset_num: &u32) -> u32 {
-    let mut result = *num;
-    if (num >= root_num && offset_num > &0) || (num < root_num && offset_num < &0) {
```

Comment:
As mentioned above, similar logic for `offset_num` is provided to remove additonal or branch